server/services/nomisPrisonerService.ts

```diff
@@ -26,31 +86,28 @@ export default class NomisPrisonerService {
 
   async getPrisonerContacts(context: Context, offenderNo: string): Promise<PrisonApiContacts> {
     const token = await this.hmppsAuthClient.getSystemClientToken(context.username)
-    logger.debug(`getting contact details for ${offenderNo}`)
     return NomisPrisonerService.restClient(token).get<PrisonApiContacts>({
       path: `/api/offenders/${offenderNo}/contacts`,
+      query: 'activeOnly=true',
     })
```

Comment:
I have a feeling the active flag in NOMIS for contacts is nonsense, best to check prod data. I'm sure we saw a prison where every contact was inactive

Reply:
oh dear, ok good idea